ext/hathach/tusb_config.h.in

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2020, Niklas Hauser
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+#include <modm/platform/device.hpp>
+
+%% for name, value in config.items() | sort
```

Comment:
But then the definitions are just moved into the modm/SConscript or cmake files. So you still have (part of) a file for this. I'd rather have this all in one place, since I'm still not entirely sure how tusb uses these rather undocumented configs (I discovered most of them through `ack "CFG_.*?"`).